data/hai/hai missions.txt

```diff
@@ -1742,3 +1742,446 @@ mission "Nanachi Meeting"
+			`	The Heliarch then start breaking it to the Hai their story, the same they told you when you first met them. While both delegations are engaged in conversation, a Heliarch agent approaches you, asking, "To this precious encounter, with what purpose have you brought these people, Captain?"`
+			choice
+				`	"They're in need of your help."`
+				`	"They come seeking technology."`
+			`	You start telling them about the Hai and their conflict with the Unfettered, but as you're explaining you hear a loud collective gasp coming from the Heliarchs that are conversing with the Hai, so you approach them once more to make sure everything is alright.`
+			`	A member of the Heliarch delegation speaks with you. "Captain <last>, kindly telling us about their territory and society, your friend Yashili has. Truthful, is it, that close to the Hai planets, a ringworld the Quarg have constructed?"`
+			`	The others seem to be in an excited conversation of their own, after turning off their translation devices. Yashili and the other Hai also turn to you, looking somewhat confused, waiting for you to respond for them.`
+			choice
+				`	"The Quarg are indeed there, but they stay out of Hai business, and the Hai stay out of theirs."`
+					goto nottheirbusiness
+				`	"Yes, the Quarg have a single system there with a complete ringworld, where many Hai and even some humans also live."`
+			`	The Heliarchs speak among themselves at a rapid pace, and before you know it several dozen other agents are around. One of them, an older Saryd with a golden circlet, says, "Tell us more about this ringworld, you must, Hai friends. Accompany us, would you?"`
+			`	Without waiting for the Hai delegation to answer, they practically drag them to a restricted section of the ringworld. About an hour later, the Hai are brought back, most with tired expressions.`
+			`	"Had I known we would be interrogated on the engineering aspects and the inner workings of a ringworld, I would have had an engineer come with us here," Yashili whispers to you.`
+			
+			label nottheirbusiness
+			`	The Heliarchs once again warn the Hai to be careful with the Quarg, and not to trust them, before finally letting the Hai say the reason for their trip here.`
+			`	The delegation still looks a bit overwhelmed, as if still processing all the history the Heliarch just told them, but Yashili has kept her composure and begins speaking. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a new ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you obtain such technology, with ship hulls that repair themselves."`
```

Comment:
```suggestion
			`	The delegation still looks a bit overwhelmed, as if still processing all the history the Heliarch just told them, but Yashili has kept her composure and begins speaking. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a new ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you possess such technology."`
```
This change is just copied and pasted from the one I suggested for the Remnant since the dialogue's the same.